### DISCUSSION TITLE
Should also list the companies that allow to easily unlock

### DISCUSSION THREAD

**Discussion by adler-amorette** [zenfyrdev/bootloader-unlock-wall-of-shame#5] [General]
Having only a list of companies that don't allow unlocking is not very useful when you're choosing what device to purchase.

While also briefly describing the process for each.

**Comment by Snuupy** [comment]
I'll start: Google Pixel devices are easy to bootloader unlock but they do not provide QFIL files to unbrick your device if it ends up in EDL mode, such as through a faulty software upgrade that Google themselves pushed out.

**Comment by zenfyrdev** [comment]
I agree that this is a great idea, but this sounds like a very short list.

- Google
- Fairphone
- Uh...

**Reply by Snuupy** [reply]
> short list

makes my shopping a lot easier then, great

**Comment by burningloww** [comment]
How about Razer? Their website implies they're good with bootloader unlocking:
https://developer.razer.com/razer-phone-dev-tools/general-instructions/

**Comment by havaianasdestruido** [comment]
- Google

- Fairphone

- Razer

- Asus (***SOME*** devices, my Zenfone Max was super easy to unlock, I only used half of a guide on XDA and did the rest alone, first time doing encrypt unlock & bootloader unlock)

**Comment by Lanchon** [comment]
# motorola

motorola does allow unlock. newer models also allow relocking. motos with officially supported grapheneos are now a reality.

they have a huge lineup and not all models are unlockable. it depends on the SoC, the carrier, etc. a simple boot into fastboot shows the "CID" value, which encodes whether a particular device can be unlocked.

there is a downside to motorola though. they force you to get an unlock code online (free and instant) but you agree to:
- a void warranty (which is probably illegal in the U.S.)
- and agreeing not to sell or transfer your unlocked device (which is probably unenforceable in the U.S. due to long held first-sale doctrine)

in all, mototola is being helpful in the technical side of things, while also being somewhat hostile in the legal side.

i much prefer that to samsung, xiaomi, oneplus, etc, but the situation is still not ideal.
